screens/Home.tsx

```diff
@@ -1,19 +1,21 @@
-import { useSQLiteContext } from "expo-sqlite";
-import React, { useState } from "react";
+import React, { useContext, useEffect, useState } from "react";
 import { Text, TextInput, StyleSheet, View, Pressable } from "react-native";
+import { TasksContext } from "../store/tasks-context";
+import type { NativeStackScreenProps } from "@react-navigation/native-stack";
 
-export default function Home({ navigation }) {
-  const db = useSQLiteContext();
-  const [inputText, setInputText] = useState("");
+type RootStackParamList = {
+  Home: undefined;
+  AllTasks: undefined;
+};
```

Comment:
`undefined` ? This is a wrong type.

Like variable names, always prefer narrower type over more broader and generic type. You should use more accurate types here, like, `AllTasks: Task[]` maybe, i don't know the specific of what data your passing, but try to get it to narrower scope.